App crashes while sharing qr code

**Summary:**

App crashes when user attempts to share the qr code of share account details

**Steps to reproduce:**

* Go to accounts -> Savings tab
* Tap on any approved account
* Click on QR Code 

**Expected behaviour:**

App shouldn't have crashed

**Stack trace**
`Caused by: android.os.BadParcelableException: ClassNotFoundException when unmarshalling: android:viewHierarchyState
        at android.os.Parcel.readParcelableCreator(Parcel.java:2847)
        at android.os.Parcel.readParcelable(Parcel.java:2768)
        at org.mifos.mobile.models.client.Client.<init>(Client.java:287)
        at org.mifos.mobile.models.client.Client$1.createFromParcel(Client.java:297)
        at org.mifos.mobile.models.client.Client$1.createFromParcel(Client.java:294)`


**Device and Android version:**

Xiaomi PocoF1
Android 9
Stock Rom

**Screen Recording:**

![GIF-200209_034455](https://user-images.githubusercontent.com/46667021/74092826-80658f00-4aef-11ea-8c3a-73fe033b766d.gif)
.

--- a/app/src/main/java/org/mifos/mobile/utils/Utils.kt
+++ b/app/src/main/java/org/mifos/mobile/utils/Utils.kt
@@ -55,7 +55,7 @@ object Utils {
         }
         val imagePath = File(context?.cacheDir, "images")
         val newFile = File(imagePath, "image.png")
-        return FileProvider.getUriForFile(context!!, "org.mifos.mobilebanking.fileprovider",
+        return FileProvider.getUriForFile(context!!, "org.mifos.mobile.fileprovider",
                 newFile)
     }
 

